nilearn/plotting/img_plotting.py

```diff
@@ -556,6 +559,163 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     return display
 
 
+def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
+                    threshold=None, linewidths=2.5, cut_coords=None,
+                    output_file=None, display_mode='ortho',
+                    figure=None, axes=None, title=None, annotate=True,
+                    draw_cross=True, black_bg='auto', dim=False,
+                    cmap=plt.cm.gist_rainbow, vmin=None, vmax=None,
+                    alpha=0.5, **kwargs):
+
+    """ Plot the multiple atlas maps or statistical maps onto the anatomical image
+        by default MNI template
+
+        Parameters
+        ----------
+        maps_img: a nifti like object or the filename of the 4D probabilistic
+            atlas maps or statistical maps
+        anat_img : Niimg-like object
+            See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
+            The anatomical image to be used as a background. If None is
+            given, nilearn tries to find a T1 template.
+        view_type: {'auto', 'contours', 'filled_contours', 'continuous'}
```

Comment:
You can go to the next line using \. This is actually the only way to please numpydoc. For an example see:
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/manifold/mds.py#L368